eleventy/shortcodes/inform-inspire.js

```diff
@@ -0,0 +1,39 @@
+import { blockPairedShortcode } from './utils.js'
+
+// Shortcode for the common visual styling around the numerous Inform Inspire
+// graphics.
+//
+// This component has 3 variants that we take into account with this shortcode:
+//
+// - No need for extra styling besides the standard container
+// - Modifiers on the content element eg: adding flex styling which we do via
+// `contentClasses`
+// - A list under the content element which we do via `list`. We handle the
+// rendering of the list here rather than in the markdown to avoid markdown
+// processing nonsense
+export const informInspire = blockPairedShortcode((content, options = {}) => {
+  const defaultOptions = {
+    classes: undefined,
+    headingLevel: 3,
+    contentClasses: undefined,
+    list: []
+  }
+  options = { ...defaultOptions, ...options }
+
+  return `<div class="app-inform-inspire${options.classes ? ` ${options.classes}` : ''}">
+    <h${options.headingLevel} class="govuk-visually-hidden">From most informative to most inspiring</h${options.headingLevel}>
+    <div class="app-inform-inspire__axis" aria-hidden="true">
+      <div class="app-inform-inspire__axis-label">Inform</div>
+      <div class="app-inform-inspire__axis-dot"></div>
+      <div class="app-inform-inspire__axis-label">Inspire</div>
+    </div>
+    <div class="app-inform-inspire__content${options.contentClasses ? ` ${options.contentClasses}` : ''}">${
+      content
+    }</div>${
+      options.list.length > 0
+        ? `<ol class="govuk-list govuk-list--number app-inform-inspire__list">
```

Comment:
I just realised that (I think) the `ol` (or previously the `ul` with `govuk-list--number`) was a leftover from when the images had numbers. I assume now that we won't use the numbers anymore that this can be just a standard `ul` (with `govuk-list--bullet`). (The PDF doesn't have any numbers here either.)

Although the numbers could still make sense because the text underneath is removed from the images to some degree and it still has a hierarchy. I'd be interested in what designers think about that: @mia-allers-gds, numbers or no numbers?
(For context, this is the list on Colour > Social.)